src/graph/graphcontext.c

```diff
@@ -557,6 +557,17 @@ static void _GraphContext_Free(void *arg) {
 	if(GraphDecodeContext_Finished(gc->decoding_context)) Graph_Free(gc->g);
 	else Graph_PartialFree(gc->g);
 
+
+	bool async_delete;
+	Config_Option_get(Config_ASYNC_DELETE, &async_delete);
+	
+	RedisModuleCtx *ctx = NULL;
+	int success = REDISMODULE_ERR;
```

Comment:
`success` unused.
What's the default value for `Config_ASYNC_DELETE` ?
Do we still feel like async deletion is valuable if we need to acquire the GIL (`RedisModule_ThreadSafeContextLock`)